src/base/bittorrent/portforwarderimpl.cpp

```diff
@@ -107,10 +107,12 @@ void PortForwarderImpl::removePorts(const QString &profile)
 
 void PortForwarderImpl::start()
 {
-    lt::settings_pack settingsPack = m_provider->get_settings();
-    settingsPack.set_bool(lt::settings_pack::enable_upnp, true);
-    settingsPack.set_bool(lt::settings_pack::enable_natpmp, true);
-    m_provider->apply_settings(settingsPack);
+    {
```

Comment:
> This is probably for hiding `settingsPack` to not leak to outside after use, right?

Sure.


> Do you mind adding a short comment in code?

I don't mind, although it should look obvious enough (in the C++ world) even without comments.

Reply:
To be honest, I can't even think of anything that doesn't look like a banal translation of the code into human language (such comments are just garbage).

Reply:
>To be honest, I can't even think of anything that doesn't look like a banal translation of the code into human language (such comments are just garbage).

It is unusual to see a new scope in non test code, so I thought it warrants a comment.
Otherwise I would just remove the curly brackets. Moving semantics is only scary when implemented badly.

Reply:
> It is unusual to see a new scope in non test code

Disagree.


> I would just remove the curly brackets.

With pleasure.

Reply:
> > I would just remove the curly brackets.
> 
> With pleasure.

Done.